packages/react-jss/src/createHoc.js

```diff
@@ -119,9 +138,9 @@ export default (stylesOrCreator, InnerComponent, options = {}) => {
       }
     }
 
-    componentDidUpdate(prevProps, prevState) {
+    componentDidUpdate(prevProps: Props, prevState: State) {
       // We remove previous dynamicSheet only after new one was created to avoid FOUC.
-      if (prevState.dynamicSheet !== this.state.dynamicSheet && prevState.dynamicSheet) {
```

Comment:
` && prevState.dynamicSheet` should not be removed!

Reply:
Same diff reason, fixed